samples/doc-snippets/src/main/java/com/example/application/AgentTeamWorkflow.java

```diff
@@ -77,8 +75,11 @@ private StepEffect askWeather() { // <2>
 
   @StepName("activities")
   private StepEffect suggestActivities() {
-    var request = // <4>currentState().userQuery +
```

Comment:
This is an example on how unrealiable is the prettier-formatter. 

The original code was:

```java
var request = currentState().userQuery +
  "\nWeather forecast: " + currentState().weatherForecast; // <4>
```

And it simply mess it up and broke it. 

We need to find a better one as this is too fragile.

Reply:
Anyway, that reveals some other issues as well.

This line broke in #862, but it was undetected. Now documentation site is failing. I can't see why it didn't fail in #862. 

And...
Note that the broken formatted code still compiles, but changes the value assigned to `request`.